### PR TITLE
Add sample schedule to instructor guide

### PR DESCRIPTION
**Pull request by javibio-git**
Initial attempt to add the sample schedule in a table to the instructor guide.

### CODE REVIEW

File: instructor_guide.Rmd [hunk 1]
```diff
@@ -1 +1,65 @@
 # Instructor Guide {-}
+
+C-MOOR miniCURE content overview.
+
+### Content overview {-}
+
+| **Name** | **Lenght** | **Notes** |
```
Comment (KatherineCox) [inline instructor_guide.Rmd:7]:
```suggestion
| **Name** | **Length** | **Notes** |
```

File: instructor_guide.Rmd [hunk 2]
```diff
@@ -1 +1,65 @@
...
+| :---: | :---: | :---: |
+| [Overview / Scientific Process](https://www.c-moor.org/miniCURE-RNA-seq/introduction.html) | 1 hr | |
+| [#1: Scientific Literature](https://www.c-moor.org/miniCURE-RNA-seq/scientific-literature.html) | Full lab | Pre-lab: Scientific Literature |
+| [#2: Model Organisms](https://www.c-moor.org/miniCURE-RNA-seq/model-organisms-and-databases.html) | Full lab | Pre-lab: Model Organisms |
+| [#3: RNA-seq Analysis](https://www.c-moor.org/miniCURE-RNA-seq/rna-seq-analysis.html) | Full lab | Pre-lab: Intro to RNA-seq |
+| [#4: Differential Gene Expression](https://www.c-moor.org/miniCURE-RNA-seq/differential-gene-expression.html) | Full lab | |
+| [#5: Kickstart Project Work](https://www.c-moor.org/miniCURE-RNA-seq/kickstart-project-work.html) | Flexible? | |
+| [Tutorial Center](https://www.c-moor.org/miniCURE-RNA-seq/tutorial-center.html) | Flexible? | Optional |
+| [Poster presentation](https://www.cloviscollege.edu/landing/stem-research-symposium-2023.html) | Full lab | Optional |
```
[the remark this answers is not in the document]
Comment (javibio-git) [reply instructor_guide.Rmd:16]: That makes sense!

**Review by KatherineCox — APPROVED** [review]

File: resources/dictionary.txt [hunk 3]
```diff
@@ -66,6 +69,8 @@ ottrpal
 padlet
 PDB
 Ph
+Photosyntesis
```
Comment (KatherineCox) [inline resources/dictionary.txt:72]:
```suggestion
Photosynthesis
```

File: instructor_guide.Rmd [hunk 4]
```diff
@@ -1 +1,66 @@
...
+#### Clovis Community College example {-}
+
+| **Spring 2023** | **Lab #** | **Lab Topic** | **miniCURE** | **Optional** |
+| :---: | :---: | :---: | :---: | :---: |
+| Jan 11 | 1 | Micropipetting           | [Overview / Scientific Process](https://www.c-moor.org/miniCURE-RNA-seq/introduction.html) |          |
+| Jan 18 | 2 | Quantitative Data Analysis | | |
+| Jan 25 | 3 | Microscopes and Cells | | |
+| Feb 1 | 4 | Diffusion and Osmosis | | |
+| Feb 8 | 5 | Enzymes | | |
+| Feb 15 | 6 | Cellular Response and Photosyntesis | Assign Onboarding | Pre-test |
```
Comment (KatherineCox) [inline instructor_guide.Rmd:34]:
```suggestion
| Feb 15 | 6 | Cellular Response and Photosynthesis | Assign Onboarding | Pre-test |
```

**Pull request merged**